book/docs/classes/OpusPool/Methods/buildUnstakeTransaction.md

```diff
@@ -0,0 +1,29 @@
+### buildUnstakeTransaction
+
+▸ **buildUnstakeTransaction**(`params`): `Promise`\<[`UnstakeTransactionData`](../../../interfaces/UnstakeTransactionData.md)\>
+
+Generates unstake transaction data to withdraw from the chosen vault
```

Comment:
I think this definition can be confusing, maybe better to say: 
 "Generates an unstake transaction data to begin the withdrawal process from the chosen vault. After signing and broadcasting, the transaction will enter the unstake queue and the amount will become withdrawable after a certain amount of time"
 It's just a suggestion, maybe @nhallenC1 can find better working.